### PR TITLE
Add receptor_log_level

### PR DESCRIPTION
**Pull request by fosterseth**
##### SUMMARY

e.g. add
`receptor_log_level: debug` to your AWX spec file

will depend on and require this AWX change https://github.com/ansible/awx/pull/14098

##### ISSUE TYPE

 - New or Enhanced Feature

### CODE REVIEW

File: config/manifests/bases/awx-operator.clusterserviceversion.yaml [hunk 1]
```diff
@@ -755,6 +755,10 @@ spec:
         x-descriptors:
         - urn:alm:descriptor:com.tectonic.ui:advanced
         - urn:alm:descriptor:com.tectonic.ui:hidden
+      - displayName: Receptor Log Level
+        path: receptor_log_level
+        x-descriptors:
+        - urn:alm:descriptor:com.tectonic.ui:text
```
Comment (fosterseth) [inline config/manifests/bases/awx-operator.clusterserviceversion.yaml]: @rooftopcellist i kind of guessed here. Is there an easy way for me to see how this gets rendered out?

**Review by TheRealHaoLiu — APPROVED** [review]

File: config/manifests/bases/awx-operator.clusterserviceversion.yaml (hunk 1 above)
Comment (rooftopcellist) [reply config/manifests/bases/awx-operator.clusterserviceversion.yaml]: You can log in to the Openshift Console, install the AWX Operator from OperatorHub, then click into that operator under "Installed Operators" and click CSV Yaml, then you can add that x-decriptor to the CSV, save it and see it show up in the form.  

This is a handy way to quickly test out new changes to the form x-descriptors.

**Review by rooftopcellist — APPROVED** [review]

**Comment by rooftopcellist** [comment]
Adding DNM to the title of this PR as @fosterseth indicated there are some changes he still needs to make.

**Comment by AlanCoding** [comment]
I merged the related change for the AWX receptor log level setting, so I took off the [DONOTMERGE] flag here. I might also restart the CI.

**Pull request merged**